packages/web/docs/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
```

Comment:
Next.js disregards this anyway, right? We could probably use `es2022` or even `esnext`.